### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Fbouchezpro**

### CODE REVIEW

**Review by SOA-P — DISMISSED** [review]

File: frontend/Desktop/app.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (LoicBrassart) [inline frontend/Desktop/app.jsx]: Don't try to create a separate Desktop app

File: frontend/src/components/Cards/Card/Index.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import "../../../services/reset.css";
```
Comment (LoicBrassart) [inline frontend/src/components/Cards/Card/Index.jsx]: Import the reset.css once (in App.jsx, or better yet in main.jsx)

File: frontend/src/components/Cards/CardsBack/Index.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import "../../../services/reset.css";
```
Comment (LoicBrassart) [inline frontend/src/components/Cards/CardsBack/Index.jsx]: See above

File: frontend/src/components/SearchButton/index.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import "@components/SearchButton/style.css";
```
Comment (LoicBrassart) [inline frontend/src/components/SearchButton/index.jsx]: -> `import "./style.css` ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
First things first: 
- Commit messages should be in English
- @Fbouchezpro will be credited for all these changes by GHub, where you all worked on this. Don't. Use. Intermediary. Branches.
- 1600+ lines, a Saturday at 2AM. I hope your code is *perfect*

File: frontend/Desktop/app.jsx (hunk 1 above)
Comment (Fbouchezpro) [reply frontend/Desktop/app.jsx]: Desktop folder delete

File: frontend/src/components/Cards/CardsBack/Index.jsx (hunk 3 above)
Comment (Fbouchezpro) [reply frontend/src/components/Cards/CardsBack/Index.jsx]: done

File: frontend/src/components/Cards/Card/Index.jsx (hunk 2 above)
Comment (Fbouchezpro) [reply frontend/src/components/Cards/Card/Index.jsx]: done

File: frontend/src/components/SearchButton/index.jsx (hunk 4 above)
Comment (Fbouchezpro) [reply frontend/src/components/SearchButton/index.jsx]: done!

**Review by AlexisFLT — APPROVED** [review]
Checked changes

**Review by LoicBrassart — APPROVED** [review]
Much better! :ok_hand:

**Pull request merged**